stable/nginx-ingress/README.md

```diff
@@ -64,6 +64,7 @@ Parameter | Description | Default
 `controller.service.externalIPs` | controller service external IP addresses | `[]`
 `controller.service.loadBalancerIP` | IP address to assign to load balancer (if supported) | `""`
 `controller.service.loadBalancerSourceRanges` | list of IP CIDRs allowed access to load balancer (if supported) | `[]`
+`controller.service.loadBalancerTerminateSSL` | terminates SSL at load balancer (forwards 443 to 80) | false
```

Comment:
Makes sense. We could make the target port configurable for both ports on the service. How about:

`controller.service.ports.http.targetPort` (default 80)
`controller.service.ports.https.targetPort` (default 443)

for the named ports "http" and "https" respectively?